schemas/view-schema.json

```diff
@@ -194,38 +194,26 @@
                 }
               },
               "readOnly": true
-            }
-          },
-          "readOnly": true
-        },
-        "focus": {
-          "type": "object",
-          "description": "Focus properties",
-          "properties": {
-            "focused_times": {
+            },
+            "in_foreground_times": {
               "type": "array",
-              "description": "List of the time the user had the document focused for browser",
+              "description": "List of the period the user had the view in foreground (focused in the browser)",
               "items": {
                 "type": "object",
-                "description": "Properties of the focused times of the view",
+                "description": "Properties of the foreground period of the view",
```

Comment:
it could be nice to use consistent wordings between the field name and the description.
- `in_foreground_times` -> a foreground time
- `in_foreground_periods` -> a foreground period
wdyt?